docker-entrypoint.sh

```
@@ -3,10 +3,10 @@
 export PROTOCOL_HEADER=x-forwarded-proto
 export HOST_HEADER=x-forwarded-host
 
-if [ -z "$BETTER_AUTH_URL" ]; then
+if [ -z "$ORIGIN" ]; then
     echo "ORIGIN is not set"
 else
-    export ORIGIN=${BETTER_AUTH_URL}
+    export ORIGIN=${ORIGIN}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)